flink-autoscaler-standalone/README.md

```diff
@@ -0,0 +1,79 @@
+# Flink Autoscaler Standalone
+
+## What's the autoscaler standalone?
+
+`Flink Autoscaler Standalone` is an implementation of `Flink Autoscaler`, it runs as 
+a separate java process. It computes the reasonable parallelism of all job vertices 
+by monitoring the metrics, such as: processing rate, busy time, etc. Please see 
+[FLIP-271](https://cwiki.apache.org/confluence/display/FLINK/FLIP-271%3A+Autoscaling) 
+for an overview of how autoscaling works.
+
+`Flink Autoscaler Standalone` rescales flink job in-place by rescale api of 
+[FLIP-291](https://cwiki.apache.org/confluence/x/9pRbDg).
```

Comment:
Here again we should link the docs instead: https://nightlies.apache.org/flink/flink-docs-release-1.18/docs/deployment/elastic_scaling/#externalized-declarative-resource-management